webapp/src/types.ts

```diff
@@ -25,3 +25,9 @@ export type RadioOptionsType = {
     id: string;
     message: string;
 }
+
+export enum Types {
+    text = 'text',
+    number = 'number',
+    textArea = 'textarea',
```

Comment:
```suggestion
export enum InputTypes {
    Text = 'text',
    Number = 'number',
    Textarea = 'textarea',
```
Also, change the name of type `InputFieldType` to `InputElementType`.